data/page-lifecycle.json

```diff
@@ -1,6 +1,6 @@
 {
-  "url": "https://github.com/WICG/web-lifecycle",
-  "title": "A Lifecycle for the Web",
+  "url": "https://wicg.github.io/page-lifecycle/spec.html",
```

Comment:
There was a bug in the Specref retrieval logic, but once #308 is merged, you can drop the `title` and `wgs` properties in that file, because they will be automatically retrieved from Specref.